includes/admin/class-wc-admin-dashboard.php

```diff
@@ -78,7 +78,7 @@ private function get_top_seller() {
 			$query['orderby'] = 'ORDER BY qty DESC';
 			$query['limits']  = 'LIMIT 1';
 
-			return $wpdb->get_row( implode( ' ', apply_filters( 'woocommerce_dashboard_status_widget_top_seller_query', $query ) ) );
+			return $wpdb->get_row( implode( ' ', apply_filters( 'woocommerce_dashboard_status_widget_top_seller_query', $query ) ) ); //phpcs:ignore
```

Comment:
Instead of just phpcs:ignore could you perhaps add the rule to ignore, just helps to avoid issues when future changes are made to this line that could cause a security issue for example.